SalesforceCommerceCloud/pwa-kit
# Edit cart item UI feedback

**GUS**: (https://gus.my.salesforce.com/a07AH000000Th7BYAS)

## How to test-drive this PR
- Checkout and run locally
- Confirm design changes
